config/routes.rb

```diff
@@ -48,4 +48,10 @@
   end
 
   root 'v0/example#index', module: 'v0'
+
+  require "sidekiq/web"
+  Sidekiq::Web.use Rack::Auth::Basic do |username, password|
+    username == ENV["SIDEKIQ_USERNAME"] && password == ENV["SIDEKIQ_PASSWORD"]
+  end if Rails.env.production?
```

Comment:
probably doesn't hurt to have this setup for dev, staging, etc as well actually as long as it has basic authentication.